golem_cli/src/command/yagna.rs

```diff
@@ -354,3 +347,56 @@ mod tracker {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
below tests will pass only after `cargo build` was invoked before and `yagna` is built in debug mode.